packaging/windows/product.wxs.in

```diff
@@ -107,12 +101,16 @@
         <SetProperty Action="CAInstallHyperv" Id="InstallHyperv" Value='"[System64Folder]dism.exe" /online /enable-feature /featureName:microsoft-hyper-v-all /NoRestart /quiet' Before="InstallHyperv" Sequence="execute"/>
         <CustomAction Id="InstallHyperv" BinaryKey="WixCA" DllEntry="WixQuietExec64" Execute="deferred" Impersonate="no" Return="ignore" />
 
+        <SetProperty Action="CAAddUserToHypervAdminGroup" Id="AddUserToHypervAdminGroup" Value='"[System64Folder]cmd.exe" /c cd /d "[INSTALLDIR]" &amp;&amp; .\crc.exe setup --log-level debug' Before="AddUserToHypervAdminGroup" Sequence="execute"/>
+        <CustomAction Id="AddUserToHypervAdminGroup" BinaryKey="WixCA" DllEntry="WixQuietExec" Execute="deferred" Impersonate="no" Return="ignore" />

```

Comment:
Does `Return="ignore"` mean that error will be ignored? How is the need for reboot handled?

Reply:
Yes, error will be ignored, we need to do that because when reboot is needed after `crc setup` the exit code is non-zero which will always make the installation fail if we don't ignore it. 

> How is the need for reboot handled?
and reboot message is always shown at the end of installation, its not smart to check if actually a reboot is needed.

Reply:
> Yes, error will be ignored, we need to do that because when reboot is needed after crc setup the exit code is non-zero which will always make the installation fail if we don't ignore it.

Ideally we'd be able to detect whether the failure was to tell us to reboot, or if this was a "real" failure we need to report back to the user.

> reboot message is always shown at the end of installation

We always tell the user to reboot, even when it's not needed? On upgrades, rebooting most likely is not required?

Reply:
> We always tell the user to reboot, even when it's not needed? On upgrades, rebooting most likely is not required?

Yes, currently even on upgrades we are asking user to reboot, there's a bug about it #2598, but during tests the reboot was needed after upgrade, if we don't reboot then `crc setup --check-only` reported error about not able to access the admin-helper named pipe.